ddtrace/contrib/httplib/patch.py

```diff
@@ -67,6 +70,17 @@ def _wrap_putrequest(func, instance, args, kwargs):
         span.set_tag(ext_http.METHOD, method)
     except Exception:
         log.debug('error applying request tags', exc_info=True)
+    return func(*args, **kwargs)
+
+
+def _wrap_putheader(func, instance, args, kwargs):
+    pin = Pin.get_from(instance)
```

Comment:
we can probably skip this since we aren't using the pin, and since we aren't creating/finishing spans here.

Reply:
no strong opinion, just an observation, not required to make this change.